diff --git a/scripts/calibrator.py b/scripts/calibrator.py
index ccdf420..d6f98da 100644
--- a/scripts/calibrator.py
+++ b/scripts/calibrator.py
@@ -433,12 +433,12 @@ if __name__ == '__main__':
         print 'Please check weights: {} existence.'.format(user_input_weights)
         sys.exit(-1)
 
-    sample = os.path.abspath(params.root + os.path.sep + 'tools/sample')
+    sample = os.path.abspath(params.root + os.path.sep + 'bin/sample')
     if not check_existence(sample):
         print 'Please check sample: {} existence.'.format(sample)
         sys.exit(-1)
 
-    caffe_bin_path = os.path.abspath(params.root + os.path.sep + 'tools/caffe')
+    caffe_bin_path = os.path.abspath(params.root + os.path.sep + 'bin/caffe')
     if not check_existence(caffe_bin_path):
         print 'Please check model/weights/sample existence.'
         sys.exit(-1)
