diff --git a/include/caffe/util/cudnn.hpp b/include/caffe/util/cudnn.hpp
index e215352..78c80ee 100644
--- a/include/caffe/util/cudnn.hpp
+++ b/include/caffe/util/cudnn.hpp
@@ -12,7 +12,7 @@
     (CUDNN_VERSION >= (major * 1000 + minor * 100 + patch))
 
 #if !defined(CUDNN_VERSION) || !CUDNN_VERSION_MIN(6, 0, 0)
-#error "NVCaffe 0.16 and higher requires CuDNN version 6.0.0 or higher"
+#warning "NVCaffe 0.16 and higher requires CuDNN version 6.0.0 or higher"
 #endif
 
 #define CUDNN_CHECK(condition) \
