diff --git a/3rdparty/cub/host/mutex.cuh b/3rdparty/cub/host/mutex.cuh
index be29d3e..829c649 100644
--- a/3rdparty/cub/host/mutex.cuh
+++ b/3rdparty/cub/host/mutex.cuh
@@ -121,9 +121,9 @@ struct Mutex
          */
         __forceinline__ void YieldProcessor()
         {
-        #ifndef __arm__
+        #ifndef __aarch64__
                 asm volatile("pause\n": : :"memory");
-        #endif  // __arm__
+        #endif  // __aarch64__
         }
 
     #endif  // defined(_MSC_VER)
